src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -270,6 +270,7 @@ function VisEditor(
 
     timefilter.enableAutoRefreshSelector();
     $scope.$listenAndDigestAsync(timefilter, 'timeUpdate', updateTimeRange);
+    $scope.$listenAndDigestAsync(timefilter, 'autoRefreshFetch', updateTimeRange);
```

Comment:
I think we won't need that. Since another auto refresh trigger, will never cause the time to actually change, since `timefilter.getTime()` will return the still stringified `from`/`to`, like "now-7d". Have you experienced any issues without that listener?